lib/axios.ts

```diff
@@ -28,8 +28,17 @@ export default async function(step: any, config) {
 }
 */
 
-export default async function(step: any, config) {
+export default async function(step: any, config, sign?) {
```

Comment:
Is sign just a boolean? It has to contain app id or something right? Should try to get a better name.

Reply:
it's an object the token and app name, i'll rename it to signConfig?